merino-adm/src/remote_settings/client.rs

```diff
@@ -482,27 +497,30 @@ mod tests {
         ChangesetResponse, RemoteSettingsAttachmentsCapability, RemoteSettingsCapabilities,
         RemoteSettingsClient, RemoteSettingsServerInfo,
     };
-    use merino_settings::providers::RemoteSettingsConfig;
+    use lazy_static::lazy_static;
     use reqwest::Url;
 
+    lazy_static! {
```

Comment:
pretty nitpicky here but static strs would do here without the need for `lazy_static`, especially with `format!` accepting them anyway (and `clone()` basically the same as `str::to_string/owned`)

`static BUCKET: &str = "main";`

Reply:
Thanks for pointing this out. I don't remember why I even did this. I remember it seemed weird at the time too.